Gems/Blast/Code/Source/Editor/EditorBlastMeshDataComponent.cpp

```diff
@@ -46,10 +46,10 @@ namespace Blast
         if (AZ::SerializeContext* serialize = azrtti_cast<AZ::SerializeContext*>(context))
         {
             serialize->Class<EditorBlastMeshDataComponent, AzToolsFramework::Components::EditorComponentBase>()
-                ->Version(4)
+                ->Version(5)
                 ->Field("Show Mesh Assets", &EditorBlastMeshDataComponent::m_showMeshAssets)
                 ->Field("Mesh Assets", &EditorBlastMeshDataComponent::m_meshAssets)
-                ->Field("Blast Slice", &EditorBlastMeshDataComponent::m_blastSliceAsset);
+                ->Field("Blast Chunks", &EditorBlastMeshDataComponent::m_blastChunksAsset);
```

Comment:
Since this is new to O3DE (it was turned off going from Lumberyard to O3DE) there will be no existing data, nor will there be a conversion from LY 1.x so there is no need for conversion this time.